File: Lab3/L3Q6.java
package Lab3;

import java.util.Scanner;

public class L3Q6 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int r, x, y; double dist;
		System.out.print("Enter the radius: ");
		r = sc.nextInt();
		System.out.print("Enter the coordinate (4 5) : ");
		x = sc.nextInt(); y = sc.nextInt(); sc.close();
		dist = Math.pow(x * x + y * y, 1 / 2);
		System.out.println(dist > r ? "Outside" : "Inside");
	}
}

File: umwix1002-TypeScript/lab/lab03/l3q6.ts
import {createConsoleInput} from "../utils/console-input";

type CoordinatePoint = [x: number, y: number]

async function l3q6Main() {
    const consoleInput = createConsoleInput();

    const radius = await consoleInput.promptInteger("Enter the radius of the circle: ");
    const x = await consoleInput.promptInteger("Enter the coordinate point (x): ");
    const y = await consoleInput.promptInteger("Enter the coordinate point (y): ");

    const center: CoordinatePoint = [0, 0]
    const distance = Math.hypot(x - center[0], y - center[1]);
    const position =
        distance === radius
            ? "on"
            : distance > radius
                ? "outside"
                : "inside";

    console.log(`The point is ${position} the circle centered at (${center[0]}, ${center[1]})`);
}

l3q6Main().catch(console.error);